docs/mobile-apps/virtual-usb.md

```diff
@@ -194,38 +199,31 @@ Run `java -jar virtual-usb-client --help` to see a full list of vUSB test config
 ### Close Test
 
 10. When you've finished testing, we recommend closing your vUSB session so that other users can use the device. There are a few ways to do this:
-  * Close the browser window in which the device is running.
-  * Close your session by running the `disconnect` command along with the `--sessionID`, which you can find in the success message.
-  ```java
-  java -jar virtual-usb-client disconnect --sessionID 37D274BC3A65A34BB3DA4DDF7B77E341
-  ```
-  Delete your sessions by running `deleteSession`, followed by your `--sessionID`, `--username`, and `--accessKey`.
-  ```java
-  java -jar virtual-usb-client deleteSession --sessionID 37D274BC3A65A34BB3DA4DDF7B77E341 --username john.smith --accessKey ab015c1e-xxxx-xxxx-xxxx-xxxxxxxxxxxx
-  ```
-  * **Android Only**: Disconnect your device from ADB by running `adb disconnect` followed by your `<IPAddress>:<portNumber>`:
-  ```java
-  adb disconnect localhost:7000
-  ```
-
-:::tip
-If you lose track of your `--sessionID`, you can recover it by running the `sessions` command, followed by your `--username` and `--accessKey`. This generates a list active device sessions:
-```java
-java -jar virtual-usb-client sessions --username john.smith --accessKey ab015c1e-xxxx-xxxx-xxxx-xxxxxxxxxxxx
-```
-:::
+   * Close the browser window where the device session is running.
+   * Close your session by running the [`disconnect`](/dev/cli/virtual-usb/disconnect-session) command along with the `--sessionId`, which you can find in the success message. If you've lost track of your `--sessionId`, recover it using the [`sessions`](dev/cli/virtual-usb/find-sessionid) command, which generates a list of your active device sessions.
+   ```java
+   java -jar virtual-usb-client disconnect --sessionId 37D274BC3A65A34BB3DA4DDF7B77E341
+   ```
+   Then, delete your session by running [`deleteSession`](/dev/cli/virtual-usb/delete-session), followed by your `--sessionId` and credentials.
```

Comment:
@kimsauce 

This is not correct, it is OR `disconnect` if you connected through a live session on Sauce, or use `deleteSession` when you started a session with `startSession`

Reply:
ooo thank you for clarifying that makes sense

Reply:
@kimsauce 
Yes, you always need to disconnect your device from ADB

Reply:
Thanks! Just added this ---

>**NOTE**: **Android Only**: After completing the above step, you'll also need to [disconnect your device from ADB](/mobile-apps/virtual-usb#close-test) by running `adb disconnect` followed by your `<IPAddress>:<portNumber>`.